circle.yml

```diff
@@ -1,23 +1,23 @@
+general:
+  artifacts:
+    - "tmp"
+
 machine:
+  environment:
+    ACCESSLINT_MASTER_BRANCH: dev
   ruby:
     version:
       2.3.1
+  node:
+    version: 6.1.0
 
 dependencies:
   pre:
-    - bundle install
-    - bundle exec jekyll build
-    # uncomment to restore pa11y-ci
-    # - nvm install stable && nvm alias default stable
-    #
-    - npm test
-    # uncomment to restore pa11y-ci
-    # - npm install -g pa11y-ci
-    #
+    - npm install -g accesslint-cli
 
 test:
-  pre:
-    - bundle exec jekyll test
+  override:
+    - npm run test
     - npm run htmlproofer
-    # uncomment to restore pa11y-ci
-    # - npm run --harmony lint-508
+  post:
+    - ./serve-accesslint && bundle exec accesslint-ci scan http://localhost:4000
```

Comment:
This `&&` chaining prevents the accesslint scan from starting before the jekyll server is launched and detached.